app/models/partners/family.rb

```diff
@@ -38,12 +39,14 @@ class Family < Base
     filterrific(
       available_filters: [
         :search_guardian_names,
-        :search_agency_guardians
+        :search_agency_guardians,
+        :include_archived
       ],
     )
 
     scope :search_guardian_names, ->(query) { where('guardian_first_name ilike ? OR guardian_last_name ilike ?', "%#{query}%", "%#{query}%") }
     scope :search_agency_guardians, ->(query) { where('case_manager ilike ?', "%#{query}%") }
+    scope :include_archived, ->(query) { query.zero? ? where(archived: false) : all }
```

Comment:
`query` in general is a bad name for this argument (I know it's following a pattern, but let's not do that now). I'd probably ensure we're passing a boolean into this argument and then write it like this:

`scope :include_archived, ->(archived) { where(archived: archived) }`

But once you have it like that, it's almost not worth making an explicit scope at all... :grin: